docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -2794,11 +2794,12 @@ Content-Type: application/json
 ]
 ```
 
-
-
 Query Parameters:
 
-- **filters** - JSON encoded value of the filters (a `map[string][]string`) to process on the networks list. Available filters: `name=[network-names]` , `id=[network-ids]`
+- **filters** - JSON encoded value of the filters (a `map[string][]string`) to process on the networks list. Available filters:
```

Comment:
It's consistent with the way it's documented for other filters though, e.g. For [list containers](https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api_v1.22.md#list-containers). Perhaps we should rewrite them all, and provide some actual examples